.travis.yml

```diff
@@ -30,8 +29,6 @@ matrix:
       env: SYMFONY_VERSION=3.1
     - php: 5.6
       env: SYMFONY_VERSION=3.2
-    - php: 5.3
```

Comment:
It [uses](https://travis-ci.org/liip/LiipMonitorBundle/jobs/188533568#L199) 2.7 because this version is declared globally and prefer lowest selects 2.7.0 instead of 2.7.latest